### PR TITLE
Use accent color for loading image animation

### PR DESCRIPTION
**Pull request by Mario54**
@lemieux @mspensieri @chloepouprom @liyefei737 @dannytranlx 

The loading animation was using the default blue color even if the accent color was set in the web app. With this change the accent color will be used if there is one.

web-loading-image

### CODE REVIEW

File: src/js/components/image-loading.jsx [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+    if (accentColor) {
+        bounceStyle.backgroundColor = `#${accentColor}`;
+    }
+
     return <div className='image-overlay'>
                <div className='three-bounce spinner'>
-                   <div className='bounce1'></div>
-                   <div className='bounce2'></div>
-                   <div className='bounce3'></div>
+                   <div className='bounce1'
```
Comment (Mario54) [inline src/js/components/image-loading.jsx]: We could refactor this part to remove the duplication, but I'm not sure it's worth the trouble?

**Comment by mspensieri** [comment]
👍

**Comment by chloepouprom** [comment]
👍

File: src/js/components/image-loading.jsx (hunk 1 above)
Comment (lemieux) [inline src/js/components/image-loading.jsx]: /shrug you're already in the image-loading component to abstract that kind of things... not sure how "optimal" it's worth going.

Comment (Mario54) [inline src/js/components/image-loading.jsx]: Yeah, it's fine as-is 😛

File: src/js/components/image-message.jsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class ImageMessage extends Component {
     render() {
         const preloader = () => {
             return <div className='preloader-container'>
-                       <ImageLoading />
+                       <ImageLoading accentColor={ this.props.accentColor } />
```
Comment (lemieux) [inline src/js/components/image-message.jsx]: just call the prop `color` ... it defines the color of the whole component

**Comment by lemieux** [comment]
:+1:

**Comment by jugarrit** [comment]
Purty

**Pull request merged**